IceCubesApp/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -158,21 +158,21 @@
 
 "settings.display.show-ipad-column" = "Zweite Spalte aktivieren";
 
-"settings.general.swipeactions" = "Swipe Actions";
-"settings.swipeactions.navigation-title" = "Swipe Actions";
-"settings.swipeactions.status.action.bookmark" = "Bookmark";
-"settings.swipeactions.status.action.boost" = "Boost";
-"settings.swipeactions.status.action.favorite" = "Favorite";
-"settings.swipeactions.status.action.none" = "None";
-"settings.swipeactions.status.action.quote" = "Quote";
-"settings.swipeactions.status.action.reply" = "Reply";
-"settings.swipeactions.status.leading" = "Swipe right";
-"settings.swipeactions.status.leading.left" = "left";
-"settings.swipeactions.status.leading.right" = "right";
-"settings.swipeactions.status.trailing" = " Swipe left";
-"settings.swipeactions.status.trailing.left" = "left";
-"settings.swipeactions.status.trailing.right" = "right";
-"settings.swipeactions.status" = "Post";
+"settings.general.swipeactions" = "Wischaktionen";
+"settings.swipeactions.navigation-title" = "Wischaktionen";
+"settings.swipeactions.status.action.bookmark" = "Lesezeichen setzen";
+"settings.swipeactions.status.action.boost" = "Boosten";
+"settings.swipeactions.status.action.favorite" = "Favorisieren";
+"settings.swipeactions.status.action.none" = "keine";
+"settings.swipeactions.status.action.quote" = "Zitieren";
+"settings.swipeactions.status.action.reply" = "Antworten";
+"settings.swipeactions.status.leading" = "Nach rechts wischen";
+"settings.swipeactions.status.leading.left" = "links";
+"settings.swipeactions.status.leading.right" = "links";
+"settings.swipeactions.status.trailing" = " Nach links wischen";
```

Comment:
I think the leading space is unnecessary. (The English one needs to be fixed too.)

Reply:
Not sure about that ... I won't remove spaces without being sure that they're not intended...?

Reply:
Yeah, I think they are not intended and should be removed.

With a space:
<img width="370" alt="SCR-20230212-it5" src="https://user-images.githubusercontent.com/56245920/218293165-400c6125-7d7a-4ea9-91a8-f26bab1bc10c.png">

Without:
<img width="368" alt="SCR-20230212-iuc" src="https://user-images.githubusercontent.com/56245920/218293177-ecbdb3cd-3ad2-4f5a-a9c1-fd0f3e5fe4cf.png">

Maybe it'll be okay to remove it when the original English string gets fixed.

Reply:
Will do … ;-)